web/src/components/jobs/Runs.tsx

```diff
@@ -0,0 +1,125 @@
+import { ArrowBackIosRounded } from '@material-ui/icons'
+import {
+  Box,
+  Chip,
+  IconButton,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Theme,
+  WithStyles,
+  createStyles,
+  withStyles
+} from '@material-ui/core'
+import { Run } from '../../types/api'
+import { alpha } from '@material-ui/core/styles'
+import { formatUpdatedAt } from '../../helpers'
+import { stopWatchDuration } from '../../helpers/time'
+import MqCode from '../core/code/MqCode'
+import MqEmpty from '../core/empty/MqEmpty'
+import MqText from '../core/text/MqText'
+import React, { FunctionComponent, SetStateAction } from 'react'
+import RunInfo from './RunInfo'
+import RunStatus from './RunStatus'
+import transitions from '@material-ui/core/styles/transitions'
+
+const RUN_COLUMNS = ['Status', 'Created At', 'Start Time', 'End Time', 'Duration']
```

Comment:
_Minor_: Can we also add the `ID` of the run? I'd also use `Started At` and `Ended At` to match the API response for [Runs](https://marquezproject.github.io/marquez/openapi.html#tag/Jobs/paths/~1jobs~1runs~1{id}/get):

```
ID, State, Started At, Ended At, Duration
```